lib/utils/playlistGenrator-utils.js

```diff
@@ -0,0 +1,1104 @@
+/**
```

Comment:
I rather if you create a folder under /lib/playlistGenerator  and not utils
+ each class in different file